third_party/externs/ts/gapi/bigquery.d.ts

```diff
@@ -60,13 +61,48 @@ declare namespace gapi.client {
       projectId: string;
     }
 
+    interface GetTableRequest {
+      datasetId: string;
+      projectId: string;
+      tableId: string;
+    }
+
     interface ListTablesResponse {
       kind: string;   // Should always be "bigquery#tableList"
       etag: string;
       nextPageToken: string;
       tables: Array<TableResource>;
     }
 
+    interface BigqueryTable {
```

Comment:
All the symbols in this file are implicitly qualified as BigTable, so I don't think that needs to be included in the interface name. Since this is the response to the GetTableRequest, I think it should be called GetTableResponse.